menpo/io/input/landmark.py

```diff
@@ -371,7 +365,7 @@ class LJSONImporter(LandmarkImporter):
 
     """
     def _parse_format(self, asset=None):
-        with open(self.filepath, 'rb') as f:
```

Comment:
@patricksnape sorry about this, think I've been bad at putting `'b'` needlessly, released this all to recently..